ECMA2Yaml/MSDNUrlPatch/UrlRepairHelper.cs

```diff
@@ -519,6 +530,12 @@ public string RemoveUnusePartFromRedirectUrl(string redirectUrl)
                 }
             }
 
+            // Locale need be removed since bug 319246 (https://ceapex.visualstudio.com/Engineering/_workitems/edit/319246)
+            if (newUrl.Contains("en-us/"))
```

Comment:
what if the url contains something like `abcden-us/`?

Reply:
updated.